es8.md

```diff
@@ -1,10 +1,10 @@
-# [Async Functions](https://github.com/lukehoban/ecmascript-asyncawait)
+This document specifies the extensions to the core ESTree AST types to support the ES8/ES2017 grammar.
```

Comment:
Is "ES8" really a term that is used? If not and if ES2017 is preferred we might _not_ want to encourage the use of "ES8" by mentioning it here. Same with the file name (although I realize that this is just following existing naming conventions).

Sorry for bikeshedding  🚲  🏠  .